include/MixerChannelView.h

```diff
@@ -58,32 +56,34 @@ class MixerChannelView : public QWidget
 	Q_PROPERTY(QColor strokeInnerActive READ strokeInnerActive WRITE setStrokeInnerActive)
 	Q_PROPERTY(QColor strokeInnerInactive READ strokeInnerInactive WRITE setStrokeInnerInactive)
 public:
+	static constexpr auto InnerBorderSize = 3;
+	static constexpr auto OuterBorderSize = 1;
```

Comment:
These can go into the implementation file as they are only used there and should hopefully never become interesting for other classes.